datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxClusterResponse.java

```diff
@@ -6,6 +6,8 @@
 
     private String name;
 
+    private SdxClusterShape shape;
```

Comment:
Any particular reason why the field is named differently than in the request?

Reply:
Sure, it should be cluster shape.
I also need to define the correct bp-cluster-template relations, so this is really a draft as of now.